src/backend/util2.c

```diff
@@ -54,6 +54,8 @@ void util_assert(const char *file, int line)
 {
     fflush(stdout);
     printf("Internal error: %s %d\n",file,line);
+    printf("This is a compiler bug, please report it via "
+        "https://issues.dlang.org/enter_bug.cgi\n");
```

Comment:
> Just an empty bug report page can be quite intimidating for a user.

If they start typing out the title they might find a duplicate one listed which could show a way to work around the bug. Even if they end up submitting a duplicate report this would get attention from bugzilla lurkers and they would get feedback pretty soon.

It would really suck if bugs like this went unreported. Bugzilla seems like a good first choice, but an ICE should be rare enough that adding two links to the diagnostics isn't such an overwhelming idea. Something like:

```
This is an unexpected internal compiler bug, please submit a bug report via https://issues.dlang.org/enter_bug.cgi?component=dmd&bug_severity=critical&keywords=ice

For more info on internal compiler errors see <this wiki page>
````